crates/uv/tests/it/edit.rs

```diff
@@ -9303,6 +9303,32 @@ fn add_index_without_trailing_slash() -> Result<()> {
     Ok(())
 }
 
+/// Add an index with a non-existent relative path.
+#[test]
+fn add_index_with_non_existent_relative_path() -> Result<()> {
```

Comment:
Can you add a test case where the `test` index exists in the project? We'll probably want to follow this up with a hint dedicated to that case.

Reply:
Added